sdk/src/lib/bitcoin/providers/provider.ts

```diff
@@ -28,7 +31,16 @@ export interface BitcoinProvider {
   getPublicKey(): Promise<string>
 
   /**
-   * Signs withdraw message.
+   * Signs withdraw message. This function is optional and if it is not
+   * implemented the `signMessage` function will be used to sign the withdrawal
+   * message.
+   * @param message Message to sign.
+   * @param data Withdrawal transaction data. @returns A signature for a given
+   * message, which proves that the owner of the account has agreed to the
+   *          message content.
```

Comment:
We're missing a line break before `@returns`.

Reply:
🤦

Reply:
Fixed in be9fa836dd5dbad164aa1d13b233ce803f529f17.